Makefile

```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 GO111MODULE = on
 
 build:
-	go build -o ./bin/status-term-client .
+	GOFLAGS=-mod=vendor go build -o ./bin/status-term-client .
```

Comment:
It won't be possible to replace it if wrote in this way. This should work the same but one can override it like this: `GOFLAGS=-some-other make build`.

```
build: GOFLAGS=-mod=vendor
build:
	go build -o ./bin/status-term-client .
.PHONY: build
```

Reply:
Oops you are right, I just did a `make build` and didn't think further. Your snippet didn't work but this seems to. Pushing fix.

```
[oskarth@localhost status-console-client]$ GOFLAGS=-some-other make build
GOFLAGS=-some-other go build -o ./bin/status-term-client .
go: parsing $GOFLAGS: unknown flag -some-other
make: *** [Makefile:7: build] Error 1
[oskarth@localhost status-console-client]$ make build
GOFLAGS="-mod=vendor" go build -o ./bin/status-term-client .
```

Reply:
N.B. My Makefile-fu isn't great, so YMMV with C/GNUMake etc